scale/scheduler/database/updater.py

```diff
@@ -143,7 +150,11 @@ def _perform_batch_field_iteration(self):
             count = cursor.rowcount
             if count:
                 logger.info('%d recipe(s) updated with batch_id %d', count, batch_id)
-            cursor.execute(qry_3, [str(batch_id), str(batch_id)])
+            cursor.execute(qry_3, [str(batch_id), str(batch_id), str(batch_id)])
+            count = cursor.rowcount
+            if count:
+                logger.info('Batch with batch_id %d set to correct recipe type revision', batch_id)
```

Comment:
Should probably have a slightly different message so it's clear which condition is being triggered if we need to debug in the future.